draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -221,70 +215,68 @@
         Another function is to offer outbound (customer to Internet) data traffic connectivity to the CAS.
       </t>
       <t>
-        The notation (AS x, {AS y1, AS y2, ...}), is used to represent an ASPA object for a CAS denoted as AS x.
-        In this notation, the set {AS y1, AS y2, ...} represents the Set of Provider ASes (SPAS) of the CAS (AS x).
```

Comment:
A set notmally uses '{' brackets. It's not critical, but I don't the reason to change it.

Reply:
I tried to use [...] for set in the ASPA and {....} for the set of ASes in the AS_PATH just to differentiate. Like said, it is not critical.

Reply:
Let's then go with {}. And let's try to avoid edits that change style but non changing meaning.

Reply:
Did any of you look at the `draft-ietf-sidrops-aspa-notation-00` and use that notation for ASPA objects?
I know that draft is still very young but it would be nice to be consistent :)

Reply:
@cjeker @mitradir
I just looked at it. They have a comma separated list of providers but no brackets around the list. We can do { } as Alexander prefers.

Reply:
@cjeker @mitradir 
Actually, we can follow the aspa-notation draft and use:
ASPA: {AS x => AS a, AS b, AS c, ...} 
The separator between the CAS and the SPAS is =>
The SPAS is simply a comma-separated set.